### DISCUSSION TITLE
the best way to get the sales estimate for a month for that particular product? identifying it by ASIN.

### DISCUSSION THREAD

**Discussion by 0xs3n4** [amzn/selling-partner-api-models#3592] [General]
What would be the best way to get the sales estimate for a month for that particular product? identifying it by ASIN.

I'm thinking of using GET_SALES_AND_TRAFFIC_REPORT, but I don't know if it's the best solution, given the vast documentation and likely options they may have.

I'm starting now with the SP-API, so if anyone can give tips I would appreciate it.

**Comment by sungolivia** [comment]
I would recommend changing the title of this post so it's more concise and to the point! That way folks can come in and provide insights!

**Reply by 0xs3n4** [reply]
Ok,I already did it, can you help me?

**Comment by 0xs3n4** [comment]
Anyone to help?

**Reply by Syazvinski** [reply]
Did you ever figure it out?

**Reply by 0xs3n4** [reply]
No, the only way I saw was to register in the event bus that tells us the daily sales of the product, and store this in its own database
